<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Showgrade</class>
 <widget class="QWidget" name="Showgrade">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>611</width>
    <height>607</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>20</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(54, 54, 54)</string>
   </property>
   <property name="text">
    <string>成绩查看及管理</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(54, 54, 54)</string>
   </property>
   <property name="text">
    <string>输入学号查询成绩：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Id">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>90</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(54, 54, 54)</string>
   </property>
   <property name="text">
    <string>输入姓名查询成绩：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="name">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>140</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Find">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>100</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_return">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>530</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>确认修改</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_return_2">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>520</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>190</y>
     <width>72</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(54, 54, 54)</string>
   </property>
   <property name="text">
    <string>学号:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>240</y>
     <width>72</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(54, 54, 54)</string>
   </property>
   <property name="text">
    <string>姓名:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="class_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>290</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="Eg">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>390</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="Cplus">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>480</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>340</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(54, 54, 54)</string>
   </property>
   <property name="text">
    <string>高数:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>430</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(54, 54, 54)</string>
   </property>
   <property name="text">
    <string>计算机综合：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>290</y>
     <width>72</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(54, 54, 54)</string>
   </property>
   <property name="text">
    <string>班级:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Id_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>190</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>480</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>C++:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>390</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(54, 54, 54)</string>
   </property>
   <property name="text">
    <string>大学英语：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Com">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>430</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="gaoshu">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>340</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="name_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>240</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>0</y>
     <width>611</width>
     <height>611</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(54, 54, 54)</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="background.qrc">:/new/prefix1/img/u=2817605393,3584994416&amp;fm=253&amp;fmt=auto&amp;app=138&amp;f=JPEG.webp</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>label_11</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>Id</zorder>
  <zorder>label_3</zorder>
  <zorder>name</zorder>
  <zorder>pushButton_Find</zorder>
  <zorder>pushButton_return</zorder>
  <zorder>pushButton_return_2</zorder>
  <zorder>label_4</zorder>
  <zorder>label_5</zorder>
  <zorder>class_2</zorder>
  <zorder>Eg</zorder>
  <zorder>Cplus</zorder>
  <zorder>label_7</zorder>
  <zorder>label_8</zorder>
  <zorder>label_6</zorder>
  <zorder>Id_2</zorder>
  <zorder>label_9</zorder>
  <zorder>label_10</zorder>
  <zorder>Com</zorder>
  <zorder>gaoshu</zorder>
  <zorder>name_2</zorder>
 </widget>
 <resources>
  <include location="background.qrc"/>
 </resources>
 <connections/>
</ui>
